XenAdmin/Wizards/NewNetworkWizard_Pages/NetWDetails.cs

```diff
@@ -144,7 +144,7 @@ private void UpdateEnablement(bool external, Host host)
             labelNIC.Visible = external;
             if (comboBoxNICList.Items.Count > 0)
                 comboBoxNICList.SelectedIndex = external ? comboBoxNICList.Items.Count - 1 : -1;
-            groupBoxSriovConfig.Visible = SelectedHostNic.sriov_physical_PIF_of.Count != 0;
+            groupBoxSriovConfig.Visible = SelectedHostNic != null && SelectedHostNic.sriov_physical_PIF_of != null && SelectedHostNic.sriov_physical_PIF_of.Count != 0;

```

Comment:
Here we should use the IsSriovPhysicalPIF() function introduced in PR #1982. We can do it in a separate PR after that one is merged.